h/static/scripts/widget-controller.js

```diff
@@ -65,15 +87,19 @@ module.exports = function WidgetController(
     return rootThread.thread(annotationUI.getState());
   }
 
+  // By default select the annotation tab
+  annotationUI.selectTab('annotation');
```

Comment:
Rather than dispatching an action to select the default tab on startup, you can just define it as part of the default state of the UI in the `initialState()` function in `annotation-ui.js`.